docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -395,6 +395,9 @@ Building with Mandrel requires a custom builder image parameter to be passed add
 ./mvnw package -Pnative -Dquarkus.native.container-build=true -Dquarkus.native.builder-image=quay.io/quarkus/ubi-quarkus-mandrel:{mandrel-flavor}
 ----
 
+Please note that the above command points to a floating tag for ease of use,
+and to make sure you get the newest Mandrel version supported by Quarkus.
+Consider using specific tags when going into production.
```

Comment:
I'm not sure I agree with:
`Consider using specific tags when going into production`. We are only going to update the floating tag, `20.1-java11`. For production it is **not recommended** to use a more specific tag, like `20.1.0.1.Final-java11`. Do I interpret this incorrectly?

Reply:
The more specific tag won't receive an update (e.g. for the base image). The `20.1-java11` tag would, though.

Reply:
@galderz @jerboaa can you guys discuss this and decide if it needs adjustments or not? I need to merge it for backporting.

Reply:
Follow up [PR](https://github.com/quarkusio/quarkus/pull/11640).